src/main/scala/com/example/Registry/ReservationSchema.scala

```diff
@@ -0,0 +1,26 @@
+package com.example.Registry
+
+import slick.jdbc.JdbcProfile
+
+final case class ReservationDB(id: Int, screeningId: Int, name: String, row: Int, seatInRow: Int)
+
+object ReservationSchema extends JdbcProfile {
+
+  import api._
+
+  class ReservationTable(tag: Tag) extends Table[ReservationDB](tag, "RESERVATIONS") {
+    def id = column[Int]("ID", O.PrimaryKey, O.AutoInc)
+
+    def screeningId = column[Int]("SCREENINGID")
+
+    def name = column[String]("NAME")
+
+    def row = column[Int]("ROW")
+
+    def seatInRow = column[Int]("SEATINROW")
```

Comment:
`SETINAROW` is hard to read. Why not snake_case for column names (and other SQL-related stuff)?